### ISSUE TITLE
 Unable to connect via SSH. Scan with -vvv option to print SSH debugging messages and check SSH settings

### ISSUE THREAD

**Issue by hktalent** [future-architect/vuls#998]
### config.ktoml
```
[servers]


[servers.ubuntu]
host         = "192.168.xxxx.xxx"
port        = "22"
user        = "root"
keyPath     = "/id_rsa"
scanMode=["fast-root"]

```

$ docker run -v ${HOME}/.ssh/id_rsa:/id_rsa -v ${HOME}/Downloads/config.toml:/vuls/config.toml --rm  vuls/vuls scan
```
time="Jun  9 02:12:33" level=info msg="Start scanning"
time="Jun  9 02:12:33" level=info msg="config: /vuls/config.toml"
time="Jun  9 02:12:33" level=info msg="Validating config..."
time="Jun  9 02:12:33" level=info msg="Detecting Server/Container OS... "
time="Jun  9 02:12:33" level=info msg="Detecting OS of servers... "
time="Jun  9 02:13:08" level=error msg="(1/1) Failed: ubuntu, err: [Failed to detect OS:\n    github.com/future-architect/vuls/scan.detectOS\n        /go/src/github.com/future-architect/vuls/scan/serverapi.go:127\n  - Unable to connect via SSH. Scan with -vvv option to print SSH debugging messages and check SSH settings. If you have never SSH to the host to be scanned, SSH to the host before scanning in order to add the HostKey. root@192.168.xxxx.xxx port: 22\n    execResult: servername: ubuntu\n      cmd: /usr/bin/ssh -tt -o StrictHostKeyChecking=yes -o LogLevel=quiet -o ConnectionAttempts=3 -o ConnectTimeout=10 -o ControlMaster=auto -o ControlPath=/root/.vuls/controlmaster-%r-ubuntu.%p -o Controlpersist=10m root@192.168.xxxx.xxx -p 22 -i /id_rsa -o PasswordAuthentication=no stty cols 1000; ls /etc/debian_version\n      exitstatus: 255\n      stdout: \n      stderr: \n      err: %!s(<nil>):\n    github.com/future-architect/vuls/scan.detectDebian\n        /go/src/github.com/future-architect/vuls/scan/debian.go:49]"
time="Jun  9 02:13:08" level=error msg="Failed to init servers: No scannable base servers:\n    github.com/future-architect/vuls/scan.InitServers\n        /go/src/github.com/future-architect/vuls/scan/serverapi.go:206"
```
<img width="573" alt="image" src="https://user-images.githubusercontent.com/18223385/84098651-63f44780-aa3a-11ea-9a19-ebac5ebea386.png">

**Comment by hktalent** [comment]
1、can not connect internet
"192.168.xxxx.xxx"
2、docker can connect internet

**Comment by MaineK00n** [comment]
Hello, I am @MaineK00n.
Thank you for reporting.

By the way, I think that the cause of this problem is that `known_hosts` is not volumed in the docker container.
The official document states as follows.

> And also, confirm that the host keys of scan target servers has been registered in the known_hosts of the localhost. To add the remote host's Host Key to $HOME/.ssh/known_hosts, you need to log in to the remote host through SSH before scanning.
ref: https://vuls.io/docs/en/tutorial-remote-scan.html

The command when using the Docker container according to the official tutorial is as follows.
By mounting `.ssh` as it is on the docker container, `known_hosts` can be passed to the docker container.
ref: https://vuls.io/docs/en/tutorial-docker.html#step6-scan
```terminal
$ docker run --rm -it \
    -v ~/.ssh:/root/.ssh:ro \
    -v $PWD:/vuls \
    -v $PWD/vuls-log:/var/log/vuls \
    -v /etc/localtime:/etc/localtime:ro \
    -e "TZ=Asia/Tokyo" \
    vuls/vuls scan \
    -config=./config.toml # path to config.toml in docker
```

> 3、in docker out, test ssh connect is ok

I understood that it is executed on the local machine, not the docker container. Is it correct?

> $ ssh -tt -o StrictHostKeyChecking=yes -o LogLevel=quiet -o ConnectionAttempts=3 -o ConnectTimeout=10 -o ControlMaster=auto -o ControlPath=/root/.vuls/controlmaster-%r-ubuntu.%p -o Controlpersist=10m  root@192.168.xxxx -p 22 -i ${HOME}/.ssh/id_rsa -o PasswordAuthentication=no stty cols 1000;ls

If you try ssh connection with `ssh -o StrictHostKeyChecking=yes`, if there is no connection destination in `known_hosts`, ssh connection should fail.
Did `~/.ssh/known_hosts` already exist in the environment where the ssh connection was verified?

I'm sorry for my poor English.
Goodbye.

**Comment by hktalent** [comment]
<img width="1001" alt="image" src="https://user-images.githubusercontent.com/18223385/84722461-dffc0b80-afb5-11ea-95ac-891d67f5d0c9.png">

config.toml
```
[servers]

[servers.ubuntu]
host         = "192.168.26.41"
port        = "22"
user        = "root"
keyPath     = "/root/.ssh/id_rsa"
scanMode=["fast-root"]
```
$ docker run --rm -it -v ${HOME}/.ssh:/root/.ssh:ro -v ${HOME}/Downloads/config.toml:/vuls/config.toml:ro  -v $PWD:/vuls   vuls/vuls scan -config=/vuls/config.toml
```
[Jun 16 01:46:03]  INFO [localhost] Start scanning
[Jun 16 01:46:03]  INFO [localhost] config: /vuls/config.toml
[Jun 16 01:46:03]  INFO [localhost] Validating config...
[Jun 16 01:46:03]  INFO [localhost] Detecting Server/Container OS...
[Jun 16 01:46:03]  INFO [localhost] Detecting OS of servers...
[Jun 16 01:46:38] ERROR [localhost] (1/1) Failed: ubuntu, err: [Failed to detect OS:
    github.com/future-architect/vuls/scan.detectOS
        /go/src/github.com/future-architect/vuls/scan/serverapi.go:127
  - Unable to connect via SSH. Scan with -vvv option to print SSH debugging messages and check SSH settings. If you have never SSH to the host to be scanned, SSH to the host before scanning in order to add the HostKey. root@192.168.26.41 port: 22
    execResult: servername: ubuntu
      cmd: /usr/bin/ssh -tt -o StrictHostKeyChecking=yes -o LogLevel=quiet -o ConnectionAttempts=3 -o ConnectTimeout=10 -o ControlMaster=auto -o ControlPath=/root/.vuls/controlmaster-%r-ubuntu.%p -o Controlpersist=10m root@192.168.26.41 -p 22 -i /root/.ssh/id_rsa -o PasswordAuthentication=no stty cols 1000; ls /etc/debian_version
      exitstatus: 255
      stdout:
      stderr:
      err: %!s(<nil>):
    github.com/future-architect/vuls/scan.detectDebian
        /go/src/github.com/future-architect/vuls/scan/debian.go:49]
[Jun 16 01:46:38] ERROR [localhost] Failed to init servers: No scannable base servers:
    github.com/future-architect/vuls/scan.InitServers
        /go/src/github.com/future-architect/vuls/scan/serverapi.go:206
```
errors, thanks @MaineK00n

**Comment by hktalent** [comment]
<img width="1242" alt="image" src="https://user-images.githubusercontent.com/18223385/84722879-fa82b480-afb6-11ea-91a3-dd03f47501e4.png">
```
$ docker run --rm -it -v ${HOME}/.ssh:/root/.ssh:ro -v $PWD:/vuls:ro   vuls/vuls scan -config=/vuls/config.toml
[Jun 16 01:53:37]  INFO [localhost] Start scanning
[Jun 16 01:53:37]  INFO [localhost] config: /vuls/config.toml
[Jun 16 01:53:37]  INFO [localhost] Validating config...
[Jun 16 01:53:37]  INFO [localhost] Detecting Server/Container OS...
[Jun 16 01:53:37]  INFO [localhost] Detecting OS of servers...
[Jun 16 01:54:12] ERROR [localhost] (1/1) Failed: ubuntu, err: [Failed to detect OS:
    github.com/future-architect/vuls/scan.detectOS
        /go/src/github.com/future-architect/vuls/scan/serverapi.go:127
  - Unable to connect via SSH. Scan with -vvv option to print SSH debugging messages and check SSH settings. If you have never SSH to the host to be scanned, SSH to the host before scanning in order to add the HostKey. root@192.168.26.41 port: 22
    execResult: servername: ubuntu
      cmd: /usr/bin/ssh -tt -o StrictHostKeyChecking=yes -o LogLevel=quiet -o ConnectionAttempts=3 -o ConnectTimeout=10 -o ControlMaster=auto -o ControlPath=/root/.vuls/controlmaster-%r-ubuntu.%p -o Controlpersist=10m root@192.168.26.41 -p 22 -i /root/.ssh/id_rsa -o PasswordAuthentication=no stty cols 1000; ls /etc/debian_version
      exitstatus: 255
      stdout:
      stderr:
      err: %!s(<nil>):
    github.com/future-architect/vuls/scan.detectDebian
        /go/src/github.com/future-architect/vuls/scan/debian.go:49]
[Jun 16 01:54:12] ERROR [localhost] Failed to init servers: No scannable base servers:
    github.com/future-architect/vuls/scan.InitServers
        /go/src/github.com/future-architect/vuls/scan/serverapi.go:206
google:vuls 0x101$ pwd
vuls
$ ls
total 8
-rwxr-xr-x  1 0x101  staff  155 Jun 16 09:52 config.toml
drwxr-xr-x  3 0x101  staff   96 Jun 16 09:44 vuls-log
```

**Comment by MaineK00n** [comment]
@hktalent

@MaineK00n, thanks for the reply.
It will be helpful to research in various environments.

# TL;DR
I want to run the following command in your environment and see the result.

```terminal
$ docker run --rm -it -v ${HOME}/.ssh:/root/.ssh:ro -v $PWD:/vuls:ro vuls/vuls scan -config=/vuls/config.toml -vvv
```

Please execute config.toml at that time under the same conditions as when you investigated.

```toml
[servers]

[servers.ubuntu]
host         = "192.168.26.41"
port        = "22"
user        = "root"
keyPath     = "/root/.ssh/id_rsa"
scanMode=["fast-root"]
```

And if it fails, I am wondering if the scan succeeds by changing config.toml as follows.

```toml
[servers]

[servers.ubuntu]
host               = "192.168.26.41"
port               = "22"
user               = "root"
sshConfigPath      = "/root/.ssh/config"
keyPath            = "/root/.ssh/id_rsa"
scanMode           =["fast-root"]
```

# Survey
From here, I will write about the results of my investigation.
First, the results in the environment shown in the figure are shown.

![IMG_0004](https://user-images.githubusercontent.com/16035056/84850313-6259fd80-b092-11ea-931b-e246ac07d62a.jpg)

```terminal
docker run --rm -it -v $HOME/.ssh:/root/.ssh:ro -v $PWD/config.toml:/vuls/config.toml -v $HOME/vuls:/vuls vuls/vuls scan -config=/vuls/config.toml
[Jun 17 00:27:47]  INFO [localhost] Start scanning
[Jun 17 00:27:47]  INFO [localhost] config: /vuls/config.toml
[Jun 17 00:27:47]  INFO [localhost] Validating config...
[Jun 17 00:27:47]  INFO [localhost] Detecting Server/Container OS...
[Jun 17 00:27:47]  INFO [localhost] Detecting OS of servers...
[Jun 17 00:27:47] ERROR [localhost] (1/1) Failed: ubuntu, err: [Failed to detect OS:
    github.com/future-architect/vuls/scan.detectOS
        /go/src/github.com/future-architect/vuls/scan/serverapi.go:115
  - Unable to connect via SSH. Scan with -vvv option to print SSH debugging messages and check SSH settings. If you have never SSH to the host to be scanned, SSH to the host before scanning in order to add the HostKey. root@172.17.0.2 port: 22
    execResult: servername: ubuntu
      cmd: /usr/bin/ssh -tt -o StrictHostKeyChecking=yes -o LogLevel=quiet -o ConnectionAttempts=3 -o ConnectTimeout=10 -o ControlMaster=auto -o ControlPath=/root/.vuls/controlmaster-%r-ubuntu.%p -o Controlpersist=10m root@172.17.0.2 -p 22 -i /root/.ssh/id_rsa -o PasswordAuthentication=no stty cols 1000; ls /etc/debian_version
      exitstatus: 255
      stdout:
      stderr:
      err: %!s(<nil>):
    github.com/future-architect/vuls/scan.detectDebian
        /go/src/github.com/future-architect/vuls/scan/debian.go:49]
[Jun 17 00:27:47] ERROR [localhost] Failed to init servers: No scannable base servers:
    github.com/future-architect/vuls/scan.InitServers
        /go/src/github.com/future-architect/vuls/scan/serverapi.go:175
```

Config.toml used at this time.

```toml
[servers]
[servers.ubuntu]
host                = "172.17.0.2"
port               = "22"
user               = "root"
keyPath            = "/root/.ssh/id_rsa"
scanMode           = ["fast-root"]
```
Here, as I asked @hktalent, I added the `-vvv` option and executed it.
Then, in my case, it turned out that `Bad owner or permissions on /root/.ssh/config` was the cause.
If @hktalent also causes the same, what you are about to write may help you.
If the cause is different, it would be helpful if you could comment the execution result.

```terminal
$ docker run --rm -it -v $HOME/.ssh:/root/.ssh:ro -v $PWD/config.toml:/vuls/config.toml -v $HOME/vuls:/vuls vuls/vuls scan -config=/vuls/config.toml -vvv
[Jun 17 01:58:10]  INFO [localhost] Start scanning
[Jun 17 01:58:10]  INFO [localhost] config: /vuls/config.toml
[Jun 17 01:58:10]  INFO [localhost] Validating config...
[Jun 17 01:58:10]  INFO [localhost] Detecting Server/Container OS...
[Jun 17 01:58:10]  INFO [localhost] Detecting OS of servers...
[Jun 17 01:58:10] ERROR [localhost] (1/1) Failed: ubuntu, err: [Failed to detect OS:
    github.com/future-architect/vuls/scan.detectOS
        /go/src/github.com/future-architect/vuls/scan/serverapi.go:115
  - Unable to connect via SSH. Scan with -vvv option to print SSH debugging messages and check SSH settings. If you have never SSH to the host to be scanned, SSH to the host before scanning in order to add the HostKey. root@172.17.0.2 port: 22
    execResult: servername: ubuntu
      cmd: /usr/bin/ssh -tt -o StrictHostKeyChecking=yes -o LogLevel=quiet -o ConnectionAttempts=3 -o ConnectTimeout=10 -o ControlMaster=auto -o ControlPath=/root/.vuls/controlmaster-%r-ubuntu.%p -o Controlpersist=10m -vvv root@172.17.0.2 -p 22 -i /root/.ssh/id_rsa -o PasswordAuthentication=no stty cols 1000; ls /etc/debian_version
      exitstatus: 255
      stdout:
      stderr: OpenSSH_8.1p1, OpenSSL 1.1.1g  21 Apr 2020
    Bad owner or permissions on /root/.ssh/config

      err: %!s(<nil>):
    github.com/future-architect/vuls/scan.detectDebian
        /go/src/github.com/future-architect/vuls/scan/debian.go:49]
[Jun 17 01:58:10] ERROR [localhost] Failed to init servers: No scannable base servers:
    github.com/future-architect/vuls/scan.InitServers
        /go/src/github.com/future-architect/vuls/scan/serverapi.go:175
```

Therefore, I changed config.toml as follows.
Added `sshConfigPath`.
This explicitly specifies the ssh config file to use when connecting with ssh.
A quick explanation of why this is needed is that the user inside the docker container running vuls
Since the UID and GID of the user on the local machine do not match, mount `.ssh` and
When I try to ssh from within the container, `Bad owner or permissions on /root/.ssh/config` occurs.
For details about `sshConfigPath`, see #986, [servers-section](https://vuls.io/docs/en/usage-settings.html#servers-section).
This is a recently merged feature, so you may need to update your vuls docker image.

```toml
[servers]
[servers.ubuntu]
host               = "172.17.0.2"
port               = "22"
user               = "root"
sshConfigPath      = "/root/.ssh/config"
keyPath            = "/root/.ssh/id_rsa"
scanMode           = ["fast"]
```

And when I execute it with the modified config.toml, the following results were obtained.

```terminal
docker run --rm -it -v $HOME/.ssh:/root/.ssh:ro -v $PWD/config.toml:/vuls/config.toml -v $HOME/vuls:/vuls vuls/vuls scan -config=/vuls/config.toml -vvv
[Jun 17 02:03:24]  INFO [localhost] Start scanning
[Jun 17 02:03:24]  INFO [localhost] config: /vuls/config.toml
[Jun 17 02:03:24]  INFO [localhost] Validating config...
[Jun 17 02:03:24]  INFO [localhost] Detecting Server/Container OS...
[Jun 17 02:03:24]  INFO [localhost] Detecting OS of servers...
[Jun 17 02:03:25]  INFO [localhost] (1/1) Detected: ubuntu: ubuntu 20.04
[Jun 17 02:03:25]  INFO [localhost] Detecting OS of containers...
[Jun 17 02:03:25]  INFO [localhost] Checking Scan Modes...
[Jun 17 02:03:25]  INFO [localhost] Detecting Platforms...
[Jun 17 02:03:25]  INFO [localhost] (1/1) ubuntu is running on other
[Jun 17 02:03:25]  INFO [localhost] Detecting IPS identifiers...
[Jun 17 02:03:25]  INFO [localhost] (1/1) ubuntu has 0 IPS integration
[Jun 17 02:03:25]  INFO [localhost] Scanning vulnerabilities...
[Jun 17 02:03:25]  INFO [localhost] Scanning vulnerable OS packages...
[Jun 17 02:03:25]  INFO [ubuntu] Scanning in fast-root mode
[Jun 17 02:03:26]  INFO [ubuntu] apt-get update...
[Jun 17 02:03:32]  WARN [ubuntu] Failed to detect a init system: File: /proc/1/exe -> /usr/bin/bash


One Line Summary
================
ubuntu  ubuntu20.04     232 installed, 6 updatable





To view the detail, vuls tui is useful.
To send a report, run vuls report -h.

```

When using docker container, please note that you cannot write to `known_hosts` because it is `:ro(read only)` if you mount it with `-v $HOME/.ssh:/root/.ssh:ro`.
See https://github.com/future-architect/vuls/issues/986#issuecomment-643834630 for details.
Copying `.ssh` somewhere (eg `~/tmp_ssh`) and doing `-v $HOME/tmp_ssh:/root/.ssh` is one easy solution.

Well, I did another survey.
As you can see in @hktalent's comment (https://github.com/future-architect/vuls/issues/998#issuecomment-640989886), I tried to scan the environment without internet connection.
It's not exactly the same as your environment, but it would be great if you could refer to it.
I'll show you what my environment is like.

![IMG_0005](https://user-images.githubusercontent.com/16035056/84850326-6a19a200-b092-11ea-839b-8233f5878c0a.jpg)

I tried running `apt update` on the SSH destination to show that the internet connection failed.
```terminal
$ ssh -i ~/.ssh/id_rsa root@192.168.11.30
The authenticity of host '192.168.11.30 (192.168.11.30)' can't be established.
ECDSA key fingerprint is SHA256:aszwu6wVcOtFUC76+HtyBX8iC9rjH11niQpTFrY/Fm0.
Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
Warning: Permanently added '192.168.11.30' (ECDSA) to the list of known hosts.
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-33-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
 * MicroK8s gets a native Windows installer and command-line integration.
     https://ubuntu.com/blog/microk8s-installers-windows-and-macos
1 update can be installed immediately.
0 of these updates are security updates.
To see these additional updates run: apt list --upgradable
Your Hardware Enablement Stack (HWE) is supported until April 2025.
The programs included with the Ubuntu system are free software;
the exact distribution terms for each program are described in the
individual files in /usr/share/doc/*/copyright.
Ubuntu comes with ABSOLUTELY NO WARRANTY, to the extent permitted by
applicable law.
root@marine:~# apt update
Err:1 http://ppa.launchpad.net/fish-shell/release-3/ubuntu focal InRelease
  Temporary failure resolving 'ppa.launchpad.net'
Err:2 http://ppa.launchpad.net/jtaylor/keepass/ubuntu focal InRelease
  Temporary failure resolving 'ppa.launchpad.net'
Err:3 https://packagecloud.io/slacktechnologies/slack/debian jessie InRelease
  Temporary failure resolving 'packagecloud.io'
Err:4 https://packages.microsoft.com/repos/ms-teams stable InRelease
  Temporary failure resolving 'packages.microsoft.com'
Err:5 https://packages.microsoft.com/repos/vscode stable InRelease
  Temporary failure resolving 'packages.microsoft.com'
Err:6 http://ppa.launchpad.net/obsproject/obs-studio/ubuntu focal InRelease
  Temporary failure resolving 'ppa.launchpad.net'
Err:7 http://storage.googleapis.com/bazel-apt stable InRelease
  Temporary failure resolving 'storage.googleapis.com'
Err:8 http://jp.archive.ubuntu.com/ubuntu focal InRelease
  Temporary failure resolving 'jp.archive.ubuntu.com'
Err:9 http://security.ubuntu.com/ubuntu focal-security InRelease
  Temporary failure resolving 'security.ubuntu.com'
Err:10 http://ppa.launchpad.net/sicklylife/filezilla/ubuntu focal InRelease
  Temporary failure resolving 'ppa.launchpad.net'
Err:11 http://dl.google.com/linux/chrome/deb stable InRelease
  Temporary failure resolving 'dl.google.com'
Err:12 http://jp.archive.ubuntu.com/ubuntu focal-updates InRelease
  Temporary failure resolving 'jp.archive.ubuntu.com'
Err:13 http://ppa.launchpad.net/thierry-f/fork-michael-gruz/ubuntu focal InRelease
  Temporary failure resolving 'ppa.launchpad.net'
Err:14 http://jp.archive.ubuntu.com/ubuntu focal-backports InRelease
  Temporary failure resolving 'jp.archive.ubuntu.com'
Reading package lists... Done
Building dependency tree
Reading state information... Done
1 package can be upgraded. Run 'apt list --upgradable' to see it.
W: Failed to fetch http://jp.archive.ubuntu.com/ubuntu/dists/focal/InRelease  Temporary failure resolving 'jp.archive.ubuntu.com'
W: Failed to fetch http://jp.archive.ubuntu.com/ubuntu/dists/focal-updates/InRelease  Temporary failure resolving 'jp.archive.ubuntu.com'
W: Failed to fetch http://jp.archive.ubuntu.com/ubuntu/dists/focal-backports/InRelease  Temporary failure resolving 'jp.archive.ubuntu.com'
W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease  Temporary failure resolving 'security.ubuntu.com'
W: Failed to fetch http://storage.googleapis.com/bazel-apt/dists/stable/InRelease  Temporary failure resolving 'storage.googleapis.com'
W: Failed to fetch http://ppa.launchpad.net/fish-shell/release-3/ubuntu/dists/focal/InRelease  Temporary failure resolving 'ppa.launchpad.net'
W: Failed to fetch http://dl.google.com/linux/chrome/deb/dists/stable/InRelease  Temporary failure resolving 'dl.google.com'
W: Failed to fetch http://ppa.launchpad.net/jtaylor/keepass/ubuntu/dists/focal/InRelease  Temporary failure resolving 'ppa.launchpad.net'
W: Failed to fetch http://ppa.launchpad.net/obsproject/obs-studio/ubuntu/dists/focal/InRelease  Temporary failure resolving 'ppa.launchpad.net'
W: Failed to fetch http://ppa.launchpad.net/sicklylife/filezilla/ubuntu/dists/focal/InRelease  Temporary failure resolving 'ppa.launchpad.net'
W: Failed to fetch https://packagecloud.io/slacktechnologies/slack/debian/dists/jessie/InRelease  Temporary failure resolving 'packagecloud.io'
W: Failed to fetch https://packages.microsoft.com/repos/ms-teams/dists/stable/InRelease  Temporary failure resolving 'packages.microsoft.com'
W: Failed to fetch http://ppa.launchpad.net/thierry-f/fork-michael-gruz/ubuntu/dists/focal/InRelease  Temporary failure resolving 'ppa.launchpad.net'
W: Failed to fetch https://packages.microsoft.com/repos/vscode/dists/stable/InRelease  Temporary failure resolving 'packages.microsoft.com'
W: Some index files failed to download. They have been ignored, or old ones used instead.
```

I will scan (`fast-root`) this environment.
The config.toml used is as follows.

```toml
[servers]

[servers.ubuntu]
host               = "192.168.11.30"
port               = "22"
user               = "root"
sshConfigPath      = "/root/.ssh/config"
keyPath            = "/root/.ssh/id_rsa"
scanMode           = ["fast-root"]
```

When I ran this config.toml, I got the following results.
The scan was successful in my environment.

```terminal
$ docker run --rm -it -v $HOME/test_ssh:/root/.ssh -v $PWD/config.toml:/vuls/config.toml -v $HOME/vuls:/vuls vuls/vuls-test scan -config=/vuls/config.toml -vvv
[Jun 17 01:32:55]  INFO [localhost] Start scanning
[Jun 17 01:32:55]  INFO [localhost] config: /vuls/config.toml
[Jun 17 01:32:55]  INFO [localhost] Validating config...
[Jun 17 01:32:55]  INFO [localhost] Detecting Server/Container OS...
[Jun 17 01:32:55]  INFO [localhost] Detecting OS of servers...
The authenticity of host '192.168.11.30 (192.168.11.30)' can't be established.
ECDSA key fingerprint is SHA256:aszwu6wVcOtFUC76+HtyBX8iC9rjH11niQpTFrY/Fm0.
Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
[Jun 17 01:33:00]  INFO [localhost] (1/1) Detected: ubuntu: ubuntu 20.04
[Jun 17 01:33:00]  INFO [localhost] Detecting OS of containers...
[Jun 17 01:33:00]  INFO [localhost] Checking Scan Modes...
[Jun 17 01:33:00]  INFO [localhost] Detecting Platforms...
[Jun 17 01:33:01]  INFO [localhost] (1/1) ubuntu is running on other
[Jun 17 01:33:01]  INFO [localhost] Detecting IPS identifiers...
[Jun 17 01:33:01]  INFO [localhost] (1/1) ubuntu has 0 IPS integration
[Jun 17 01:33:01]  INFO [localhost] Scanning vulnerabilities...
[Jun 17 01:33:01]  INFO [localhost] Scanning vulnerable OS packages...
[Jun 17 01:33:01]  INFO [ubuntu] Scanning in fast-root mode
[Jun 17 01:33:03]  INFO [ubuntu] apt-get update...


One Line Summary
================
ubuntu  ubuntu20.04     2466 installed, 1 updatable




To view the detail, vuls tui is useful.
To send a report, run vuls report -h.
```

I hope you can help me.
Thank you for keeping me company with my poor English.

**Comment by kotakanbe** [comment]
Re-open if you're still having problems
